extras/wordgrinder.mime

```diff
@@ -0,0 +1,2 @@
+application/wordgrinder
```

Comment:
Should probably be `application/x-wordgrinder` --- I'm not registered with IANA...